entry/src/main/resources/base/element/string.json

```diff
@@ -0,0 +1,16 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
```

Comment:
Remove the unnecessary strings and replace the required strings with meaningful string values.

Reply:
Done